presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -1626,6 +1632,11 @@ public static boolean useMarkDistinct(Session session)
         return session.getSystemProperty(USE_MARK_DISTINCT, Boolean.class);
     }
 
+    public static boolean exploitConstraints(Session session)
```

Comment:
nits: Let's keep the naming consistent: `isExploitConstraints`

Reply:
Done.